CHANGELOG.md

```diff
@@ -1,3 +1,94 @@
+### citus v11.2.0 (January 30, 2023) ###
+
+* Adds support for `MERGE` command on local tables
+
+* Adds support for outer joins having recurring tuples in the outer side of the
+  join (e.g., \<reference table\> LEFT JOIN \<distributed table\>)
+
+* Adds support for creating `PRIMARY KEY`s and `UNIQUE`/`EXCLUSION`/`CHECK`/
+  `FOREIGN KEY` constraints via `ALTER TABLE` command without providing a
+  constraint name
+
+* Adds support for using identity columns on Citus managed tables
```

Comment:
We seem to support identity columns for all tables. Citus managed tables is mostly used to refer citus local tables (e.g., "citus.use_citus_managed_tables",).

Maybe just say: Adds support for using identity columns